presets/4.3/tune/Tehllama_533_TinyTrainer_3in_3S_TUNE_Options.txt

```diff
@@ -0,0 +1,135 @@
+#$ TITLE: FlyFive33 3in TinyTrainer 3S Tune
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: TinyTrainer, 533, Spec, Race, Llama
+#$ AUTHOR: Daniel Appel / Tehllama 
+#$ DESCRIPTION: Tune developed by Armando 'Mondo' Gallegos and Evan 'HeadsUp' Turner
+#$ DESCRIPTION: This is an adaptation of the 533 TinyTrainer tune, and expects bidirectional DShot ESCs at 48kHz PWM with MedHigh timing. 
+#$ DESCRIPTION: These fly best on 3S batteries in the 450-550mAh capacity range with 533 1404 motors.
+#$ DISCLAIMER: This tune requires ESC with Bidirectional DShot Capable Firmware, and works best at 48kHz PWM and MedHigh or 23° Motor Timing.     By default, this overwrites the current tune profile and makes TPA changes in the current rateprofile.
```

Comment:
extra spaces

Reply:
Finally corrected - also added option groups to simplify process, and corrected verbiage about ESC requirements (now that I have tested without RPMF - the all-SDFT version is solid).